src/Data/Nat/Properties.agda

```diff
@@ -1185,6 +1190,46 @@ m⊔n≤m+n m n with ⊔-sel m n
 ... | inj₁ m⊔n≡m rewrite m⊔n≡m = m≤m+n m n
 ... | inj₂ m⊔n≡n rewrite m⊔n≡n = m≤n+m n m
 
+------------------------------------------------------------------------
+-- Other properties of _⊔_ and _*_
+
+*-distribˡ-⊔ : _*_ DistributesOverˡ _⊔_
+*-distribˡ-⊔ m zero o = begin-equality
+  m * (zero ⊔ o)                                        ≡⟨⟩
+  m * o                                                 ≡⟨⟩
+  zero ⊔ m * o                                          ≡˘⟨ cong (_⊔ m * o) (*-zeroʳ m) ⟩
```

Comment:
I think it's probably overkill to expand out the cases that don't require multiple steps. This case can probably just be `cong (_⊔ m * o) (*-zeroʳ m)`.